walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiableCredential.kt

```diff
@@ -12,7 +13,7 @@ import java.util.Date
 @Serializable
 internal class VerifiableCredential(
     internal val raw: VerifiableCredential,
-    internal val contract: VerifiableCredentialContract,
+    internal val contract: VerifiableCredentialContract? = null,
```

Comment:
I think this was introduced by one of my PRs but we decided to not make it nullable.

Reply:
I don't think I ever changed this back in my changes. Does this need to be changed back now?